packages/react/src/components/Storage/FileUploader/UploadButton/index.tsx

```diff
@@ -26,6 +26,7 @@ export function UploadButton({
       <VisuallyHidden>
         <input
           type="file"
+          tabIndex={-1}
```

Comment:
why do we want to "Made sure input is not tab-able"?

Reply:
Because the control of the input is handled by the button. We don't want keyboard users to access the input directly or screen readers to announce the input or those users might get confused with 2 focusable elements. 

<img width="636" alt="Screen Shot 2022-11-01 at 2 51 44 PM" src="https://user-images.githubusercontent.com/321279/201507240-88006b64-e38d-4f66-acf0-b233b97e09e2.png">